build.gradle

```diff
@@ -1,173 +1,175 @@
-  compile "org.apache.mesos:mesos:${mesosVer}"
-  compile "org.apache.commons:commons-lang3:3.4"
-  compile "org.slf4j:log4j-over-slf4j:${slf4jVer}"
-  compile "org.slf4j:jcl-over-slf4j:${slf4jVer}"
-  compile "com.googlecode.protobuf-java-format:protobuf-java-format:${protobufFormatVer}"
-  compile "org.json:json:${jsonVer}"
-  compile "com.google.inject:guice:${guiceVer}"
-  compile "com.google.inject.extensions:guice-assistedinject:${guiceVer}"
-  testCompile "com.github.stefanbirkner:system-rules:${systemRulesVer}"
-  testCompile "io.dropwizard:dropwizard-testing:${dropwizardVer}"
-  testCompile "org.mockito:mockito-all:${mockitoVer}"
-  testCompile "org.powermock:powermock-mockito-release-full:${powerMockVer}"
-  testCompile "org.mock-server:mockserver-netty:${mockServerVer}"
-  testCompile "org.springframework.integration:spring-integration-http:${springVer}"
-  testCompile "org.awaitility:awaitility:${awaitilityVer}"
+    compile "com.fasterxml.jackson.datatype:jackson-datatype-guava:${jacksonVer}"
+    compile "com.fasterxml.jackson.datatype:jackson-datatype-jdk8:${jacksonVer}"
+    compile "com.fasterxml.jackson.datatype:jackson-datatype-jsr310:${jacksonVer}"
+    compile "com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:${jacksonVer}"
+    compile "com.fasterxml.jackson.core:jackson-databind:${jacksonVer}"
+    compile "com.kenai.nbpwr:edu-umd-cs-findbugs-annotations:${findbugsAnnotateVer}"
+    compile "commons-collections:commons-collections:${commonsCollectionsVer}"
+    compile "commons-io:commons-io:${commonsIoVer}"
+    compile "org.antlr:antlr4-runtime:${antlrVer}"
+    compile "javax.ws.rs:javax.ws.rs-api:${restServiceVer}"
+    compile "org.apache.curator:curator-framework:${curatorVer}"
+    compile "org.apache.curator:curator-recipes:${curatorVer}"
+    compile "org.apache.curator:curator-test:${curatorVer}"
+    compile "org.apache.httpcomponents:httpclient:${httpClientVer}"
+    compile "org.apache.httpcomponents:fluent-hc:${httpClientVer}"
+    compile "org.apache.mesos:mesos:${mesosVer}"
+    compile "org.apache.commons:commons-lang3:3.4"
+    compile "org.slf4j:log4j-over-slf4j:${slf4jVer}"
+    compile "org.slf4j:jcl-over-slf4j:${slf4jVer}"
+    compile "com.googlecode.protobuf-java-format:protobuf-java-format:${protobufFormatVer}"
+    compile "org.json:json:${jsonVer}"
+    compile "com.google.inject:guice:${guiceVer}"
+    compile "com.google.inject.extensions:guice-assistedinject:${guiceVer}"
+    compile "org.glassfish.jersey.containers:jersey-container-jetty-http:2.23.2"
+    testCompile "com.github.stefanbirkner:system-rules:${systemRulesVer}"
+    testCompile "io.dropwizard:dropwizard-testing:${dropwizardVer}"
+    testCompile "org.mockito:mockito-all:${mockitoVer}"
+    testCompile "org.powermock:powermock-mockito-release-full:${powerMockVer}"
+    testCompile "org.mock-server:mockserver-netty:${mockServerVer}"
+    testCompile "org.springframework.integration:spring-integration-http:${springVer}"
```

Comment:
Unrelated to this PR, but we should remove this dependency on `Spring`. It brings in many transitive dependencies. It's only being use in a Test class: https://github.com/mesosphere/dcos-commons/blob/master/src/test/java/org/apache/mesos/dcos/MockDcosCluster.java#L76

It can be replaced with https://jersey.java.net/apidocs/2.7/jersey/javax/ws/rs/core/MediaType.html#APPLICATION_JSON_TYPE (already a dependency on the project).